README.md

```diff
@@ -36,6 +36,70 @@ GitHub Action to run Renovate self-hosted.
 | <a href="https://renovatebot.com"><img alt="Renovate enabled" src="https://img.shields.io/badge/renovate-enabled-brightgreen.svg?style=flat-square"></a>                                                                | Dependencies | Renovate             |
 | <a href="https://github.com/renovatebot/github-action/actions"><img alt="GitHub workflow status" src="https://img.shields.io/github/actions/workflow/status/renovatebot/github-action/build.yml?style=flat-square"></a> | Build        | GitHub Actions       |
 
+## Quick Start
```

Comment:
As noted, it might be worth adding the word `GitHub` here -- possibly with other words (`Workflow`, "using a"?)

```suggestion
## Quick Start with a GitHub Workflow 
```